app/docker/components/log-viewer/logViewer.html

```diff
@@ -62,7 +72,7 @@
 
 <div class="row" style="height:54%;">
   <div class="col-sm-12" style="height:100%;">
-    <pre class="log_viewer" scroll-glue="$ctrl.state.autoScroll" force-glue>
+    <pre ng-class="$ctrl.getLogViewerCSSClass()" scroll-glue="$ctrl.state.autoScroll" force-glue>
```

Comment:
Instead of using a function, I'd rather see an object here:

```html
<pre ng-class="{ 'wrap_lines': $ctrl.state.wrapLines }" class="log_viewer" ...>
</pre>
```

Reply:
fixed in 78dde81b